src/yuzu/configuration/configure_input_player.cpp

```diff
@@ -102,6 +102,16 @@ QString GetButtonName(Common::Input::ButtonNames button_name) {
         return QObject::tr("Share");
     case Common::Input::ButtonNames::Options:
         return QObject::tr("Options");
+    case Common::Input::ButtonNames::ButtonMouseWheel:
+        return QObject::tr("Wheel");
```

Comment:
```suggestion
        return QObject::tr("Wheel", "Indicates the mouse wheel");
```

Since `"Wheel"` can be quite ambiguous in isolation to a translator